automation/src/test/scala/org/broadinstitute/dsde/test/api/AuthDomainGroupRoleSpec.scala

```diff
@@ -61,7 +61,7 @@ class AuthDomainGroupRoleSpec extends AnyFreeSpec with WorkspaceFixtures with Gr
             AuthDomainMatcher.checkVisibleAndAccessible(projectName, workspaceName, List(groupName))(studentToken)
 
             // remove "student" from billing project
-            Rawls.billing.removeUserFromBillingProject(projectName, student.email, BillingProjectRole.Owner)(
+            Rawls.billingV2.removeUserFromBillingProject(projectName, student.email, BillingProjectRole.Owner)(
```

Comment:
This is because the newer version of `workbenchLibs` no longer has the v1 code. The revert of the v1 API deletion reintroduced `Rawls.billing` calls in this file.